lib/bank_account/saving_account.rb

```diff
@@ -0,0 +1,30 @@
+module BankAccount
+  class SavingAccount < Account
+
+    def initialize(id, initial_balance)
```

Comment:
The comment's great, but if there's no difference between the two, you don't even have to include `initialize` here. Inheritance will bring the base class's intializer into this class. :)